docs/release_notes/november_2023.md

```diff
@@ -0,0 +1,24 @@
+# November 2023
+## airbyte v0.50.34 to v0.50.35
+
+This page includes new features and improvements to the Airbyte Cloud and Airbyte Open Source platforms.
+
+## ✨ Highlights
+
+Airbyte now supports extracting text content from PDF, Docx, and Pptx files from S3, Azure Blob Storage, and the newly introduced [Google Drive](/integrations/sources/google-drive.md) source. This is an important part of supporting LLM use use cases that rely on unstructured data in files.
+
+SSO and RBAC (admin roles only) are now available in Airbyte Cloud! Read more below.
+
+## Platform Releases
+- **SSO and RBAC** You can now use SSO in Airbyte Cloud to administer permissions in Airbyte. This is currently only available through Okta, with plans to support Active Directory next. We also now offer **RBAC** (admin roles only) to ensure a high level of security when managing you workspace. For access to this feature, reach out to our [Sales team](https://www.airbyte.com/company/talk-to-sales).
+- **Continuous heartbeat checks** We're continually monitoring syncs to ensure they continue making progress, and have added functionality in the background to ensure that we continue receiving updated ["heartbeat" messages](/understanding-airbyte/heartbeats.md) from our connectors. This will ensure that we continue delivering data and avoid any timeouts. 
+
+## Connector Improvements
+
+In addition to being able to extract text content from unstructured data sources, we have also:
+
+ - Revamped core Marketing connectors Pinterest, Instagram and Klaviyo to significantly improve the experience in setting up and ensure resiliency and reliability.
```

Comment:
- Revamped core Marketing connectors Klaviyo, Pinterest, and Instagram to significantly improve the setup experience and ensure resiliency and reliability.

Reply:
+1